source/includes/fact-older-version-limitations.rst

```diff
@@ -0,0 +1,42 @@
+- Writes that produce DDL events cannot occur on the source cluster during the 
```

Comment:
should this PR be based against `mongodb:feature-older-version-support` feature branch instead?

Reply:
a h, thanks for catching that — fixed.